apps/fz_http/lib/fz_http_web/controllers/auth_controller.ex

```diff
@@ -67,6 +67,16 @@ defmodule FzHttpWeb.AuthController do
          {:ok, claims} <- openid_connect.verify(provider, tokens["id_token"]) do
       case UserFromAuth.find_or_create(provider, claims) do
         {:ok, user} ->
+          # only first-time connect will include refresh token
```

Comment:
Hmm thinking through this, if Firezone is offline for an extended time, will that disable all OIDC users permanently? Or will it just force them to re-authenticate at the next sign-in?